### PR TITLE
Closes User story 4 - Receive Payments

### LINKED ISSUE #4: 4-receive-payments
As a band,
So that I can fund my music making,
I would like to be able to receive payments.

### PR DESCRIPTION
**Pull request by MarcoBgn**
closes #4

### CODE REVIEW

File: back-end/spec/controllers/charges_controller_spec.rb [hunk 1]
```diff
@@ -36,124 +36,5 @@
   # ChargesController. Be sure to keep this updated too.
   let(:valid_session) { {} }
```
Comment (ghost) [inline back-end/spec/controllers/charges_controller_spec.rb:11]: Extra line.

File: back-end/spec/spec_helper.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require "rails_helper"
+require "payments_helper"
 # This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (ghost) [inline back-end/spec/spec_helper.rb]: Comments

File: back-end/app/controllers/charges_controller.rb [hunk 3]
```diff
@@ -3,26 +3,27 @@ class ChargesController < ApplicationController
...
+    charge = Stripe::Charge.create(
+      customer: customer.id, 
+      amount: @amount,
+      description: 'artist contribution',
+      currency: 'GBP'
+      ) 
+      
+      redirect_to 'http://localhost:9000/#/success'
           
     rescue Stripe::CardError => e
```
Comment (ghost) [inline back-end/app/controllers/charges_controller.rb]: is e a convention?

File: back-end/spec/rails_helper.rb [hunk 4]
```diff
@@ -5,7 +5,8 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'spec_helper'
 require 'rspec/rails'
-require 'rspec/capybara'
+require 'capybara/rails'
+# require 'selenium-webdriver'
 # Add additional requires below this line. Rails is not loaded until this point!
```
Comment (ghost) [inline back-end/spec/rails_helper.rb]: comments

File: front-end/app/views/about.html [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
     src="https://checkout.stripe.com/checkout.js" class="stripe-button"
     data-key="pk_test_6pRNASCoBOKtIshFeQd4XMUh"
     data-image="/img/documentation/checkout/marketplace.png"
     data-name="Stripe.com"
     data-description="2 widgets"
-    data-amount="2000"
+    data-amount=""
     data-locale="auto">
   </script>
 </form>
```
Comment (yanyil) [inline front-end/app/views/about.html]: empty lines

File: front-end/test/spec/controllers/about.js [hunk 6]
```diff
@@ -15,7 +15,4 @@ describe('Controller: AboutCtrl', function () {
     });
   }));
```
Comment (ghost) [inline front-end/test/spec/controllers/about.js:17]: Extra line

**Comment by ghost** [comment]
:shipit:

**Pull request merged**